### ISSUE TITLE
CanJS Getting Started Video

### ISSUE THREAD

**Issue by justinbmeyer** [canjs/canjs#4168]
tldr; Make a new "Getting Started with CanJS" video to replace the [old one](https://www.youtube.com/watch?v=GdT4Oq6ZQ68).  The video will be about 45 minutes and cover as much critical information we can cover in that time.

## Problem

People like learning from video.  It's more engaging and can teach people faster than reading.  Also, sites with video tend to convert more people.

The [old getting started video](https://www.youtube.com/watch?v=GdT4Oq6ZQ68) was made for CanJS 1.1.  But it had over 30k views.  

It's type for a new video.

## Solution 

I propose we create a new "Getting Started With CanJS" video for CanJS 5.0. The video has two primary goals:

- Get people interested in CanJS
- Give people enough knowledge to start using CanJS

The content will be a mix of what's on the homepage and the [5.0 technology overview](https://canjs.github.io/next/doc/guides/technology-overview.html).

**Comment by justinbmeyer** [comment]
- [ ] In many packages?
